lib/ansible/modules/cloud/amazon/iam_cert.py

```diff
@@ -96,9 +95,17 @@
     cert: body_of_somecert
     key: vault_body_of_privcertkey
     cert_chain: body_of_myverytrustedchain
+
+# Basic rename of existing certificate
+- iam_cert:
+    name: very_ssl
+    new_name: new_very_ssl
+    state: present
+
 '''
-import json
-import sys
+from ansible.module_utils.basic import *
```

Comment:
should probably address this to at some point. But I can always fix this after.

Usually 

```
from ansible.module_utils.basic import AnsibleModule
from ansible.module_utils.ec2 import ec2_argument_spec, get_aws_connection_info, connect_to_aws
```
suffices but flake8 will tell you if you've missed anything.